dev_guide/volumes.adoc

```diff
@@ -23,14 +23,14 @@ are present, to repair them when possible, {product-title} invokes the `fsck`
 utility prior to the `mount` utility. This occurs when either adding a volume or
 updating an existing volume.
 
-The simplest volume type is `*EmptyDir*`, which is a temporary directory on a
+The simplest volume type is `EmptyDir`, which is a temporary directory on a
 single machine. Administrators may also allow you to request a
 xref:persistent_volumes.adoc#dev-guide-persistent-volumes[persistent volume] that is automatically attached
 to your pods.
 
 [NOTE]
 ====
-`*EmptyDir*` volume storage may be restricted by a quota based on the pods FSGroup, if enabled by your cluster administrator.
+`EmptyDir` volume storage may be restricted by a quota based on the pods FSGroup, if enabled by your cluster administrator.
```

Comment:
s/pods/pod's
s/if enabled/if the FSGroup parameter
Do we need a link to enable FSGroup?